datafusion/datasource/src/file_scan_config.rs

```diff
@@ -440,7 +441,6 @@ impl FileScanConfigBuilder {
             Statistics::new_unknown(file_source.table_schema().file_schema())
```

Comment:
Shouldn't this use the complete table schema (with the partition columns too) ?
```suggestion
            Statistics::new_unknown(file_source.table_schema().table_schema())
```